app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,2 @@
+<%= render :template => 'kaui/layouts/kaui_application' %>
```

Comment:
I believe this will generate incorrect HTML: `kaui_application` renders the full HTML page (`<html></html>`) and the additional JS file will be included after the closing `</html>`. This might work because browsers are lenient.

Maybe we could add a config in killbill-admin-ui to inject additional headers in https://github.com/killbill/killbill-admin-ui/blob/master/app/views/kaui/layouts/kaui_header.html.erb, e.g.:

```
...
  <title>Kaui</title>
  <%= stylesheet_link_tag 'kaui_application', :media => 'all' %>

  <% unless Kaui.additional_headers_partial.nil? %>
    <%= render :partial => Kaui.additional_headers_partial %>
  <% end %>

  <%= csrf_meta_tags %>
...
```

and from here, we configure `additional_headers_partial`?